.env.sample

```diff
@@ -2,3 +2,19 @@
 
 # Preview
 # EXPERIMENTER_API_CALL="experiments/?status=Preview&application=firefox-desktop"
+
+# Auth0 properties
+# A long, secret value used to encrypt the session cookie
```

Comment:
Please add the thing from elsewhere in the PR about how to use `openssl` to generate this.